src/data/objects/profile.yaml

```diff
@@ -50,7 +50,7 @@ schema:
       value: 2
       description: Total number of referrals attributed to user that have not yet converted to full accounts.
     credit:
-      type: Integer
+      type: Float
       value: 160
```

Comment:
if we're doing 40.0 as the example for longviewsubscription.yaml, shouldn't we also add the ".0" at the end of all of these other US dollar float value examples as well?